src/auth/login/Login.jsx

```diff
@@ -94,7 +94,7 @@ const Login = () => {
                   <MailSvg />
                   <FormInput
                     type="text"
-                    placeholder="ayooluwakunle@gmil.com"
+                    placeholder="ayooluwakunle@gmail.com"
```

Comment:
@beblicarl let's add  [test to the component](https://testing-library.com/docs/react-testing-library/intro/)